### ISSUE TITLE
关于torch与paddle的对应api

### ISSUE THREAD

**Issue by lxk767363331** [PaddlePaddle/Paddle#26339]
avgout = torch.mean(x, dim=1, keepdim=True)=== avg_out = fluid.layers.reduce_mean(input, dim=1, keep_dim=True)

 maxout, _ = torch.max(x, dim=1, keepdim=True)===max_out = fluid.layers.reduce_max(input, dim=1, keep_dim=True)

  out = self.ca(out) * out  # 广播机制=====conv=ChannelAttention(conv,num_filters=num_filters,ratio=4)*conv
  out = self.sa(out) * out  # **广播机制**=====  conv=SpatialAttention(conv,num_filters=num_filters,)*conv

torch中代码对应地方api我这样设置但是有些问题

**Comment by lijianshe02** [comment]
请详细描述您的问题并提供您的代码

**Comment by lxk767363331** [comment]
问题是torch中的api与paddle中api我这样的对应关系是否正确
torch.mean= fluid.layers.reduce_mean
torch.max=fluid.layers.reduce_max
在飞浆paddlepaddle公众号的文章中给的是这样的api但是我觉得不对
torch.mean = fluid.layers.mean
torch.max = fluid.layers.argmax
文章链接：https://mp.weixin.qq.com/s/wl9T_Go5V7I_SLDO-xBdNw
代码位置：文章最末尾代码的Spatial Attention Module位置

**Comment by nikankind** [comment]
torch.max()实际实现了多个功能，如：
torch.max(Tensor)求Tensor的最大值
torch.max(Tensor,dim)求Tensor在某维度下的若干个最大值
torch.max(Tensor1,Tensor2)把两个张量相应位置依次比较，并输出每个位置的最大值

fluid.layers.reduce_max()可以实现torch.max()前两个功能

同理，torch.min()对应fluid.layers.reduce_min()

给的两版API对照表格文件在此处有点纰漏
